bin/docker-entrypoint.sh

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
+set -e
+
+if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
+	if [ -n "$MYSQL_PORT_3306_TCP" ]; then
+		if [ -z "$WORDPRESS_DB_HOST" ]; then
+			WORDPRESS_DB_HOST='mysql'
+		else
+			echo >&2 'warning: both WORDPRESS_DB_HOST and MYSQL_PORT_3306_TCP found'
+			echo >&2 "  Connecting to WORDPRESS_DB_HOST ($WORDPRESS_DB_HOST)"
+			echo >&2 '  instead of the linked mysql container'
+		fi
+	fi
+
+	if [ -z "$WORDPRESS_DB_HOST" ]; then
+		echo >&2 'error: missing WORDPRESS_DB_HOST and MYSQL_PORT_3306_TCP environment variables'
+		echo >&2 '  Did you forget to --link some_mysql_container:mysql or set an external db'
+		echo >&2 '  with -e WORDPRESS_DB_HOST=hostname:port?'
+		exit 1
+	fi
+
+	# if we're linked to MySQL and thus have credentials already, let's use them
+	: ${WORDPRESS_DB_USER:=${MYSQL_ENV_MYSQL_USER:-root}}
+	if [ "$WORDPRESS_DB_USER" = 'root' ]; then
```

Comment:
I'm pretty sure we want to get this from Consul, and not support injection via env vars.